### PR TITLE
Chore/update readme

### PR DESCRIPTION
**Pull request by Ignaciomendezp**
# Version 1.0.10 🎉

### Context

This is a [Soyio package](https://www.npmjs.com/package/@soyio/soyio-widget?activeTab=readme) for web applications used for registration and authentication of identities.

### What is being done

Updated readme
​

---

#### Additional Info (screenshots, links, sources, etc.)

---

#### Before you merge...

> [!IMPORTANT]
> - [x] **Version Update**: Confirm that the `package.json` has been updated to reflect a new version that is higher than the current version on the [main branch](https://github.com/Soyio-id/soy-io-widget), which should be the same version that is available on [npm](https://www.npmjs.com/package/@soyio/soyio-widget).This step is crucial as it allows for an automatic release process.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -161,7 +161,12 @@ The `onEvent` callback is designed to handle various events that occur during wi
   - `identityId`: The unique identifier for the authenticated identity.
   - `userReference`: The reference identifier for the user, facilitating the association of the event with the user within the company's context.
 
-- **`DENIED_CAMERA_PERMISSION`**: Event triggered when user denies camera permissions. It closes the widget.
+- **`DENIED_CAMERA_PERMISSION`**: Event triggered when user clicks de "reject" button in the signature attempt. The event object includes:
```
Comment (jmaldonadof) [inline README.md:164]: Están intercambiados los eventos o no? Este es cámara y describe rechazo de la firma.

File: README.md [hunk 2]
```diff
@@ -161,7 +161,12 @@ The `onEvent` callback is designed to handle various events that occur during wi
...
+
+  - `identityId`: The unique identifier for the identity.
+  - `userReference`: The userReference used in the validation attempt for the identity.
+
+- **`REJECTED_SIGNATURE`**: Event triggered when user denies camera permissions. It closes the widget.
```
Comment (jmaldonadof) [inline README.md:169]: Y este es rechazo de firma y describe lo de los permisos de la cámara.

**Review by jmaldonadof — CHANGES_REQUESTED** [review]
Te comenté porque creo que hay un error.

File: README.md (hunk 1 above)
Comment (Ignaciomendezp) [reply README.md:164]: Toda la razón 😅

**Review by jmaldonadof — APPROVED** [review]
👍🏼

**Pull request merged**